tests/integration/Cargo.toml

```diff
@@ -19,3 +19,4 @@ serde = { workspace = true, features = ['derive'] }
 control-panel-api = { path = '../../core/control-panel/api', version = '0.0.2-alpha.2' }
 upgrader-api = { path = '../../core/upgrader/api', version = '0.0.2-alpha.2' }
 station-api = { path = '../../core/station/api', version = '0.0.2-alpha.2' }
+wat = "1.0.52"
```

Comment:
ideally this should be in the workspace root Cargo.toml and then this inner Cargo.toml would just need `wat = { workspace = true }`